spec/03-plugins/35-azure-functions/01-access_spec.lua

```diff
@@ -38,11 +98,22 @@ for _, strategy in helpers.each_strategy() do
         },
       }
 
-      assert(helpers.start_kong{
-        database = strategy,
-        plugins  = "azure-functions",
+      local fixtures = {
+        dns_mock = helpers.dns_mock.new()
+      }
+
+      fixtures.dns_mock:A({
+        name = "mockbin.org",
```

Comment:
```suggestion
        name = "azure.example.com",
```

Maybe use a different name (other than `mockbin.org`) to not confuse future readers.

Reply:
done